command/allocations/app.go

```
@@ -70,7 +70,8 @@ func NewFirehose() (*Firehose, error) {
 
 	sink, err := sink.GetSink()
 	if err != nil {
-		return nil, err
```

this *should* work, since the CLI tool will exit 1 when an error is returned upstream to `main.go`

---

should is great word, 
i added it cause it failed silently